Packs/PAN-OS/ReleaseNotes/2_1_10.md

```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### PAN-OS - Block Destination Service
+
+Added the `SecondaryDeviceGroup` input in the case that we want to add the block rule to the "Shared" location put still push to applicable device-groups.
```

Comment:
```suggestion
Added the `SecondaryDeviceGroup` input in case we want to add the block rule to the "Shared" location put still push to applicable device groups.
```